graphql_compiler/schema_transformation/rename_schema.py

```diff
@@ -542,15 +548,22 @@ def _rename_or_suppress_or_ignore_name_and_add_to_record(
         if (
             new_name_string in self.reverse_name_map
             and self.reverse_name_map[new_name_string] != name_string
-        ):
+        ) or new_name_string in self.custom_scalar_types:
+            # Either new_name_string is the name of a type (that may or may not have been
+            # renamed) in reverse_name_map or new_name_string is the name of a custom scalar.
+            # Either way, we want the original name of the type that is would be named
+            # new_name_string in the renamed schema.
```

Comment:
Can you add one line here about why `.get()` is the correct call on the next line? I'm sure working through the complex logic in the conditional, one can convince themselves that is the case, but it's nice to optimize the code for readability and help the reader out with such logical leaps.